manifests/4.7/local-storage-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -102,6 +102,7 @@ metadata:
       Configure and use local storage volumes in kubernetes and OpenShift.
       OpenShift 4.2 and above is only supported OpenShift versions.
     olm.skipRange: ">=4.3.0 <4.7.0"
+    operators.operatorframework.io/internal-objects: '["localvolumediscoveryresults.local.storage.openshift.io"]' 
```

Comment:
for completeness sake, can you also update opm-bundle directory's CSV?

Reply:
This has been included.